tests/upgrades/test_repository.py

```diff
@@ -143,29 +137,25 @@ def test_pre_scenario_custom_repo_check(self, target_sat, save_test_data):
         lce = target_sat.api.LifecycleEnvironment(organization=org).create()
 
         product = target_sat.api.Product(organization=org).create()
-        target_sat.create_custom_html_repo(rpm1)
-        repo = target_sat.api.Repository(
-            product=product.id, url=f'{target_sat.url}/pub/custom_repo'
-        ).create()
+        repo = target_sat.api.Repository(product=product.id, url=settings.repos.yum_1.url).create()
         repo.sync()
-
-        content_view = publish_content_view(org=org, repolist=repo)
+        content_view = target_sat.publish_content_view(org, repo)
         content_view.version[0].promote(data={'environment_ids': lce.id})
-
-        subscription = target_sat.api.Subscription(organization=org).search(
-            query={'search': f'name={product.name}'}
-        )[0]
         ak = target_sat.api.ActivationKey(
             content_view=content_view, organization=org.id, environment=lce
         ).create()
-        ak.add_subscriptions(data={'subscription_id': subscription.id})
+        if target_sat.api.Organization(id=org.id).read().simple_content_access is False:
+            subscription = target_sat.api.Subscription(organization=org).search(
+                query={'search': f'name={product.name}'}
+            )[0]
+            ak.add_subscriptions(data={'subscription_id': subscription.id})
         rhel7_client = Broker(container_host='ubi7:latest', host_class=ContentHost).checkout()
```

Comment:
can we update it to use rhel8 client instead as we've all el8 now? non-blocking

Reply:
I will move this to rhel8 by default thanks for reminding me